doc/main.html

```diff
@@ -789,6 +789,39 @@ <h4>Lists of key-value pairs</h4>
 
       </section>
 
+      <section id="sec-list-element">
+        <h4>Lists</h4>
+
+        <p>A list can be inserted using either an <code>ol</code> or <code>ul</code> element. The <code>ul</code> (unordered list) element should be used when the order of listed items is not meaniful or important. The <code>ol</code> (ordered list) element should be only used when the order of listed items is meaningful and/or important to the list and/or document.</p>
+
+        <p>An <code>ol</code> or <code>ul</code> element must contain:</p>
```

Comment:
Don't we want to say that an `ol` or `ul` shall contain one or more `li` element, and each `li` element can contain one or more `ol` or `ul` elements?

Reply:
True, will update.

Reply:
Done in https://github.com/SMPTE/html-pub/pull/155/commits/176216373b289729496240bf05a0c5f7cb41a821